lib/onc_certification_g10_test_kit/onc_program_procedure.yml

```diff
@@ -745,28 +745,41 @@ procedure:
           [Both] The health IT developer demonstrates the ability of the Health
           IT Module to receive the following access token request parameters
           from an application according to the implementation specification
-          adopted in § 170.215(a)(3):
+          adopted in § 170.215(c)(2):
+
           * “grant_type”;
           * “code”;
           * “redirect_uri”;
           * “code_verifier”;
-          * “client_id”; and
-          * Authorization header including “client_id” and “client_secret”.
+          * “client_id” (to support “client-public” “SMART on FHIR®
+            Capability”);
+          * Authorization header including “client_id” and “client_secret” (to
+            support “client-confidential-symmetric” “SMART on FHIR®
+            Capability”); and
+          * Authentication JSON Web Token (to support
+            “client-confidential-asymmetric” “SMART on FHIR® Capability”)
         TLV: |
           [Both] The tester verifies the ability of the Health IT Module to
           receive the following access token request parameters from an
           application according to the implementation specification adopted in §
-          170.215(a)(3):
+          170.215(c)(2):
+
           * “grant_type”;
           * “code”;
           * “redirect_uri”;
           * “code_verifier”;
-          * “client_id”; and
-          * Authorization header including “client_id” and “client_secret”.
+          * “client_id” (to support “client-public” “SMART on FHIR®
+            Capability”);
+          * Authorization header including “client_id” and “client_secret” (to
+            support “client-confidential-symmetric” “SMART on FHIR®
+            Capability”); and
+          * Authentication JSON Web Token (to support
+            “client-confidential-asymmetric” “SMART on FHIR® Capability”)
         inferno_supported: 'yes'
         inferno_tests:
-          - 1.3.05
```

Comment:
Should we remove reference to SMART v1 test since this test apply to both SMART v1 and v2.

Reply:
`AUT-PAT-30` is SMART v2 only.